app/subscriber/src/features/my-folders/ConfigureFolder.tsx

```diff
@@ -41,7 +41,6 @@ export const ConfigureFolder: React.FC<IConfigureFolderProps> = ({ active }) =>
 
   const [activeFilter, setActiveFilter] = React.useState<IFilterModel>();
   const [actionName, setActionName] = React.useState<'delete' | 'empty'>();
-  const [currentFolder, setCurrentFolder] = React.useState<IFolderModel>();
```

Comment:
This was the state variable becoming out of sync, and we don't really need it with context provider